devel/kcov/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        SimonKagstrom kcov 37 v
+github.tarball_from archive
+
+categories          devel
+license             GPL-2
+platforms           darwin
+maintainers         nomaintainer
+
+description         coverage report generator
+long_description    \
+    Kcov is a code coverage tester for compiled programs, Python scripts and \
+    shell scripts. It allows collecting code coverage information from \
+    executables without special command-line arguments, and continuously \
+    produces output from long-running applications.
+homepage            https://simonkagstrom.github.com/kcov/index.html
```

Comment:
This is the `homepage` generated by the GitHub portgroup. So this redundant line should be removed.

Reply:
I believe the autogenerated home page is https://github.com/SimonKagstrom/kcov which is more of a kcov developer landing page than the included URL.

Reply:
OK. Weird, I've never seen that type of homepage URL before.